### PR TITLE
Release v4.1.2

### PR DESCRIPTION
**Pull request by strigazi**
Signed-off-by: Spyros Trigazis <spyridon.trigazis@cern.ch>

**Note**: oauth2client is now deprecated. As such, it is unlikely that we will
review or merge to your pull request. We recommend you use
[google-auth](https://google-auth.readthedocs.io) and [oauthlib](http://oauthlib.readthedocs.io/).

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,14 @@
 # CHANGELOG
 
+## v4.1.2
+
+**Note**: oauth2client is deprecated. No more features will be added to the
+libraries and the core team is turning down support. We recommend you use
+[google-auth](https://google-auth.readthedocs.io) and [oauthlib](http://oauthlib.readthedocs.io/).
+
+Bug fixes:
+* Fix conflicts on site-packages/tests. (#688)
```
Comment (theacodes) [inline CHANGELOG.md]: Suggestion:

```
* Fix packaging issue hat erroneously installed the test package. (#688)
```

**Comment by strigazi** [comment]
@jonparrott Looks good to you?

File: CHANGELOG.md (hunk 1 above)
Comment (strigazi) [reply CHANGELOG.md]: DONE

Comment (theacodes) [reply CHANGELOG.md]: I'm not seeing the updated commit?

Comment (strigazi) [reply CHANGELOG.md]: done

Comment (strigazi) [reply CHANGELOG.md]: You meant, "had erroneously", right?

**Review by theacodes — APPROVED** [review]

**Comment by strigazi** [comment]
@jonparrott The new version goes to pypi automatically? Do we need an extra step?

**Comment by jonparrott** [comment]
Travis is supposed to do it automatically, but it looks like travis' deployment plugin is using the deprecated PyPI endpoint and they just started a brownout((log)[https://travis-ci.org/google/oauth2client/jobs/248497987#L909]).

/cc @dstufft

**Comment by dstufft** [comment]
See https://github.com/travis-ci/dpl/issues/655

**Comment by jonparrott** [comment]
Thanks, @dstufft, you are incredibly helpful as always. :)

**Comment by strigazi** [comment]
Thanks, @dstufft 

@jonparrott It looks like that 4.1.2 will be available tomorrow (at some timezone :-) ). 
Is there anything that I can do to help configure the new endpoint?

**Comment by jonparrott** [comment]
@strigazi I just need to do a manual release. I'll try to do so today.

**Comment by jonparrott** [comment]
@strigazi it's published on PyPI: https://pypi.python.org/pypi/oauth2client

**Pull request merged**